.windows-10-sdk-version

```diff
@@ -0,0 +1 @@
+20348
```

Comment:
There are later versions than this, but we know this works in CI. We can try newer versions later on.